src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -113,20 +112,20 @@ public void equals() {
         // different types -> returns false
         assertFalse(modelManager.equals(5));
 
-        // different addressBook -> returns false
-        assertFalse(modelManager.equals(new ModelManager(differentAddressBook, userPrefs)));
+        // different gradPad -> returns false
+        assertFalse(modelManager.equals(new ModelManager(differentGradPad, userPrefs)));
 
         // different filteredList -> returns false
-        String[] keywords = ALICE.getName().fullName.split("\\s+");
-        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(keywords)));
-        assertFalse(modelManager.equals(new ModelManager(addressBook, userPrefs)));
+        String[] keywords = CS2103T.getModuleCode().moduleCode.split("\\s+");
```

Comment:
Maybe we can remove the regex split? It doesn't do anything since module codes never have spaces in them. Maybe we can just create an array with the module code in it as a single string:

```suggestion
        String[] keywords = { CS2103T.getModuleCode().moduleCode };
```

Reply:
Oh okay yeah sounds good.